apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -1152,7 +1153,8 @@ namespace MWRender
     {
         mPlayerAnimation = new NpcAnimation(player, player.getRefData().getBaseNode(), mResourceSystem, 0, NpcAnimation::VM_Normal,
                                                 mFirstPersonFieldOfView);
-
+        if(mPlayerNode->getNumParents()==0)
+            mObjects->getOrCreateCell(player.getCell())->addChild(mPlayerNode);
```

Comment:
This line causes a crash (null pointer dereferencing) for me. Player's cell can and will be null before the game is fully loaded. How did it get past your own testing?

Reply:
newgame was a case i haven't checked..I added a fix...it seems to do the trick